contracts/pods/PodFactory.sol

```diff
@@ -0,0 +1,225 @@
+
+    ///////////////////// GETTERS ///////////////////////
+
+    /// @notice Get the address of the Gnosis safe that represents a pod
+    /// @param podId Unique id for the orca pod
+    function getPodSafe(uint256 podId) public view override returns (address) {
+        return podController.podIdToSafe(podId);
+    }
+
+    /// @notice Get the number of pod members
+    /// @param podId Unique id for the orca pod
+    function getNumMembers(uint256 podId)
+        external
+        view
+        override
+        returns (uint256)
+    {
+        address safe = getPodSafe(podId);
+        address[] memory members = IGnosisSafe(safe).getOwners();
+        return uint256(members.length);
+    }
+
+    /// @notice Get all members on the pod
+    /// @param podId Unique id for the orca pod
+    function getPodMembers(uint256 podId)
+        public
+        view
+        override
+        returns (address[] memory)
+    {
+        address safeAddress = podController.podIdToSafe(podId);
+        return IGnosisSafe(safeAddress).getOwners();
+    }
+
+    /// @notice Get the signer threshold on the pod
+    /// @param podId Unique id for the orca pod
+    function getPodThreshold(uint256 podId)
+        external
+        view
+        override
+        returns (uint256)
+    {
+        address safe = getPodSafe(podId);
+        uint256 threshold = uint256(IGnosisSafe(safe).getThreshold());
+        return threshold;
+    }
+
+    /// @notice Get the next pod id
+    function getNextPodId() external view override returns (uint256) {
+        return memberToken.getNextAvailablePodId();
+    }
+
+    /// @notice Get the podAdmin from the base Orca controller
+    /// @dev Controller only allows existing admin to change
+    /// @param podId Unique id for the orca pod
+    function getPodAdmin(uint256 podId)
+        external
+        view
+        override
+        returns (address)
+    {
+        return IControllerV1(podController).podAdmin(podId);
+    }
+
+    //////////////////// STATE-CHANGING API ////////////////////
+
+    /// @notice Create a child Orca pod with optimistic timelock. Callable by the DAO and the Tribal Council
+    /// @param _members List of members to be added to the pod
+    /// @param _threshold Number of members that need to approve a transaction on the Gnosis safe
+    /// @param _podLabel Metadata, Human readable label for the pod
+    /// @param _ensString Metadata, ENS name of the pod
+    /// @param _imageUrl Metadata, URL to a image to represent the pod in frontends
+    /// @param _minDelay Delay on the timelock
+    /// @param _podAdmin The admin of the pod - able to add and remove pod members
+    function createChildOptimisticPod(
+        address[] calldata _members,
+        uint256 _threshold,
+        bytes32 _podLabel,
+        string calldata _ensString,
+        string calldata _imageUrl,
+        uint256 _minDelay,
+        address _podAdmin
+    ) external override onlyOwner returns (uint256, address) {
```

Comment:
who is the owner of this contract? imo this should use TribeRoles right?

Reply:
I was looking for a way to be able to call this function, with null members, via the deploy address in the first step of deploying the contract and not require a DAO action to do it (makes the DAO script much simpler). 

Have transitioned to TribeRoles, but have a concept of a `deployer` that is able to call the createPod fn. Can discuss

Reply:
Update: adding a burner function for initial deploy